smithy-swift-codegen/build.gradle.kts

```diff
@@ -17,6 +17,10 @@ extra["moduleName"] = "software.amazon.smithy.swift.codegen"
 group = "software.amazon.smithy"
 version = "0.1.0"
 
+configurations.implementation {
+    exclude(group = "brazil")
+}
```

Comment:
**Question**: Why is this necessary?

Reply:
Required just for time being until dependencies are pushed to maven.